onur/styles.css

```diff
@@ -0,0 +1,241 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    color:#18181a;
+}
+html {
+    font-family: "Open Sans", sans-serif;
+}
+/* navbar section */
+a {
+    text-decoration: none;
+    color: #363636;
+    font-size: 1rem;
+}
+.navbar ul li a:hover {
+    text-decoration:underline;
+    color: black;
+}
+
+.text-center {
+    text-align: center;
+}
+.heading-small {
+    font-size: 1.2rem;
+    font-weight: 400;
+    text-align: left;
+}
+
+.btn {
+    color:rgb(30, 143, 218);
+    border: 1px solid rgb(30, 143, 218);
+    border-radius: 5px;
+    padding: 0.6rem;    
+}
+.btn:hover {
+    background-color: rgb(30, 143, 218);
+    color:white;
+}
+.btn:active {
+    box-shadow: 0 0 2px 2px #3e8fc5;
+}
+.conteiner {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1.25rem;
+    background: rgb(255, 255, 255);
+    height: 70px;
+    border-bottom: 1px solid rgb(222 226 230);
+    box-shadow: rgba(0, 0, 0, 0.05) 0px 7px 11px 0px;  
+}
+.navbar ul {
+    display: flex;
+    align-items: center;
+    gap: 1.2rem; 
+    list-style-type: none;
+}
+/* description section */
+.description {
+    background: (255, 255, 255);
+    max-width: 700px;
+    width: 70%;
+    height: 200px;
+    margin-top: 4rem;
+    margin-left: auto;
+    margin-right: auto;
+}
+.description h1 {
+    margin-bottom: 0.6rem;
+    font-size: 3.3rem;
+    font-weight: 200;
+    opacity: 0.9;
+}
+.description p {
+    letter-spacing: 0.7px;
+    line-height: 1.7rem;
+    opacity: 0.9;
+    color:#363636;
+    font-size: 1.1rem;
+    
+}
+/* content section */
+.total-content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 1rem;
+}
+.content {  
+    margin-bottom: 4rem;
+    display: flex;
+    flex-direction: row;
+    gap: 1.8rem;
+    justify-content: center;
+    align-items: flex-start;
+    flex-wrap: wrap;
+}
+.prices {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+.head {
+    width: 18rem;
```

Comment:
I'd say if you set width explicitly for the card, you should only do so for the entire card (the most outer div of the card) and let all the other content be 100%. A better approach would be to align cards with CSS and not set the width explicitly at all (let it be as wide as content needs to) depending on the available space on the screen. It may sense to specify the min-width property though as it would make sure that the card does not get too small.